File: U3JAVA/U3P_10.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u3p_10;
import java.util.Scanner;
/**
 *
 * @author alfre
 */
public class U3P_10 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Ingrese las horas trabajadas: ");
        int horasTrabajadas = sc.nextInt();

        System.out.print("Ingrese el pago por hora: ");
        double pagoPorHora = sc.nextDouble();

        double sueldoSemanal;

        if (horasTrabajadas <= 40) {
            sueldoSemanal = horasTrabajadas * pagoPorHora;
        } else {
            // Las primeras 40 horas se pagan normalmente
            sueldoSemanal = 40 * pagoPorHora;

            // Horas extras se pagan al doble
            int horasExtras = horasTrabajadas - 40;
            sueldoSemanal += horasExtras * (pagoPorHora * 2);
        }

        System.out.println("El sueldo semanal es: $" + sueldoSemanal);
    }
}

File: U3C#/U3P_10.cs
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class U3P_10
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese las horas trabajadas: ");
            int horasTrabajadas = Convert.ToInt32(Console.ReadLine());

            Console.Write("Ingrese el pago por hora: ");
            double pagoPorHora = Convert.ToDouble(Console.ReadLine());

            double sueldoSemanal;

            if (horasTrabajadas <= 40)
            {
                sueldoSemanal = horasTrabajadas * pagoPorHora;
            }
            else
            {
                // Las primeras 40 horas se pagan normalmente
                sueldoSemanal = 40 * pagoPorHora;

                // Horas extras se pagan al doble
                int horasExtras = horasTrabajadas - 40;
                sueldoSemanal += horasExtras * (pagoPorHora * 2);
            }

            Console.WriteLine("El sueldo semanal es: $" + sueldoSemanal);
        }
    }
}